### PR TITLE
Create Work, About me and Contacts section

### PR DESCRIPTION
**Pull request by johnspeny**
**Work, About, Contacts Section**

1. Created work section with grid and flex
2. About section has flex box with skills
3. Contacts section has forms and social links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <span>
+                    <button>See Project</button>
+                </span>
+            </div>
+        </div>
```
Comment (Emmyn5600) [inline index.html:122]: - [ ] The paragraph (e.g., "A daily section ...") is a p tag, Please include the paragraph as it is in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: index.html [hunk 2]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                <h2>Multi-Post Stories</h2>
+                <h2>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <span>
+                    <button>See Project</button>
```
Comment (Emmyn5600) [inline index.html:63]: - [ ] The paragraph (e.g., "A daily section ...") is a p tag, Please include the paragraph as it is in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: index.html [hunk 3]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                </span>
+            </div>
```
Comment (Emmyn5600) [inline index.html:79]: - [ ] - [ ] The paragraph (e.g., "A daily section ...") is a p tag, Please include the paragraph as it is in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: index.html [hunk 4]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                <h2>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <span>
+                    <button>See Project</button>
+                </span>
```
Comment (Emmyn5600) [inline index.html:92]: - [ ] The paragraph (e.g., "A daily section ...") is a p tag, Please include the paragraph as it is in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: index.html [hunk 5]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                <h2>Multi-Post Stories</h2>
+                <h2>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <span>
+                    <button>See Project</button>
```
Comment (Emmyn5600) [inline index.html:105]: - [ ] The paragraph (e.g., "A daily section ...") is a p tag, Please include the paragraph as it is in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: index.html [hunk 6]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
```
Comment (Emmyn5600) [inline index.html:119]: - [ ] The paragraph (e.g., "A daily section ...") is a p tag, Please include the paragraph as it is in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: style.css [hunk 7]
```diff
@@ -74,3 +74,140 @@ body {
...
+}
+
+.item {
+  border: 1px solid #dfe1e6;
+  text-align: center;
+  padding: 16px;
+  border-radius: 5px;
+  margin-bottom: 50px;
+}
+
```
Comment (Emmyn5600) [inline style.css:97]: - [ ] Original
orig

- [ ] Yours
your

* Kindly, Fix that to make yours look the same as the original, for more refer to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: index.html [hunk 8]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+            <h1 class="headline__nameOne">About me</h1>
+        </div>
+        <div class="headline__bodyText">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <span>
+            <button>Get My Resume</button>
+        </span>
```
Comment (Emmyn5600) [inline index.html:138]: - [ ] Original
about

- [ ] Yours
yours

* Kindly, Fix that to make yours look the same as the original, for more refer to the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

File: index.html [hunk 9]
```diff
@@ -32,5 +32,173 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                <span><input type="submit" value="Get in touch"></span>
+            </form>
+        
+            <ul class="headline__social">
+                <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i>
+                </a></li>
+                <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>        
+            </ul>
+        <h3>johnspeny@gmail.com</h3>
```
Comment (Emmyn5600) [inline index.html:199]: - [ ] Original
contac

- [ ] Yours
contactyour

* Please fix this as well.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
#### Hi @johnspeny :raised_hands:

Good job so far!:clap::clap:
We, however, have some issues to look into before we may move forward but you are almost there!:muscle:

##  Required Changes :recycle::warning: 

* Please you need commits oftentimes instead of pushing all changes at once
* Please follow the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) to implement the changes requested.

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by johnspeny** [comment]
@Emmyn5600 I am using template 3 from Figma. Please have a look at it.

File: index.html [hunk 10]
```diff
@@ -32,5 +32,180 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                <h2>Gain+Glory</h2>
+                <div class="work__language">
+                    <p>Ruby on rails</p>
+                    <p>css</p>
+                    <p>JavaScript</p>
+                    <p>html</p>
+                </div>
+                <span>
+                    <button>See Project</button>
+                </span>
```
Comment (Emmyn5600) [inline index.html:96]: - [ ] Original
proj

- [ ] Yours

proj2
Hint:
* resize your image to fit into the card
* Make margin-top to your button

File: index.html [hunk 11]
```diff
@@ -32,5 +32,180 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
+                <h2>Multi-Post Stories</h2>
+                <h2>Gain+Glory</h2>
+                <div class="work__language">
+                    <p>Ruby on rails</p>
+                    <p>css</p>
+                    <p>JavaScript</p>
+                    <p>html</p>
+                </div>
+                <span>
+                    <button>See Project</button>
```
Comment (Emmyn5600) [inline index.html:125]: - [ ] Original
proj

- [ ] Yours

proj2
Hint:
* resize your image to fit into the card
* Make margin-top to your button

File: index.html [hunk 12]
```diff
@@ -32,5 +32,180 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
```
Comment (Emmyn5600) [inline index.html:95]: - [ ] Original
proj

- [ ] Yours

proj2
Hint:
* resize your image to fit into the card
* Make margin-top to your button

File: index.html [hunk 13]
```diff
@@ -32,5 +32,180 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
```
Comment (Emmyn5600) [inline index.html:80]: - [ ] Original
proj

- [ ] Yours

proj2
Hint:
* resize your image to fit into the card
* Make margin-top to your button

File: index.html [hunk 14]
```diff
@@ -32,5 +32,180 @@ <h1 class="headline__nameOne">Hey there. I'm Okker John.</h1>
...
```
Comment (Emmyn5600) [inline index.html:65]: - [ ] Original
proj

- [ ] Yours

proj2
Hint:
* resize your image to fit into the card
* Make margin-top to your button

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
#### Hi @johnspeny :raised_hands:

Good job so far!:clap::clap:
We, however, have some issues to look into before we may move forward but you are almost there!:muscle:

##  Required Changes :recycle::warning: 

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by ahcodes — APPROVED** [review]

**Pull request merged**